// juniper.afi_cap_match is generated by proto_generator as a protobuf
// representation of a YANG schema.
//
// Input schema modules:
//  - afi-cap-action.yang
//  - afi-cap-entry-action.yang
//  - afi-cap-entry-match.yang
//  - afi-cap-entry.yang
//  - afi-cap-match.yang
//  - afi-cap.yang
//  - afi-common.yang
//  - afi-cos-classifier.yang
//  - afi-counter.yang
//  - afi-decap.yang
//  - afi-discard.yang
//  - afi-encap-entry.yang
//  - afi-encap.yang
//  - afi-filter.yang
//  - afi-fwd-sandbox.yang
//  - afi-indexed-list.yang
//  - afi-indirect.yang
//  - afi-list.yang
//  - afi-match.yang
//  - afi-oam.yang
//  - afi-policer.yang
//  - afi-port.yang
//  - afi-receive.yang
//  - afi-replicate.yang
//  - afi-sample.yang
//  - afi-sandbox.yang
//  - afi-selector.yang
//  - afi-switch.yang
//  - afi-table.yang
//  - afi-tree-encap-entry.yang
//  - afi-tree-encap.yang
//  - afi-tree-entry.yang
//  - afi-tree.yang
//  - afi-types.yang
//  - afi.yang
syntax = "proto3";

package juniper.afi_cap_match;

import "ywrapper/ywrapper.proto";
import "yext/yext.proto";

message AfiCapMatch {
  ywrapper.BoolValue arp_target_ipv4_address = 183979878;
  ywrapper.BoolValue destination_ipv4_address = 4420347;
  ywrapper.BoolValue destination_mac_address = 354602281;
  ywrapper.BoolValue destination_port = 378515304;
  ywrapper.BoolValue ethertype = 282240232;
  ywrapper.BoolValue icmp_type = 136750768;
  ywrapper.BoolValue ingress_class_id = 378178964;
  ywrapper.BoolValue inner_ip_proto = 214803407;
  ywrapper.BoolValue inner_tpid = 471227898;
  ywrapper.BoolValue inner_vlan_cfi = 127058409;
  ywrapper.BoolValue inner_vlan_dot1p = 510272471;
  ywrapper.BoolValue inner_vlan_id = 21049916;
  ywrapper.BoolValue ip_checksum_valid = 25197184;
  ywrapper.BoolValue ip_protocol = 278078440;
  ywrapper.BoolValue ipv4_ttl = 188153678;
  ywrapper.BoolValue l3_interface_class = 461536750;
  ywrapper.BoolValue l4_destination_port = 10026449;
  ywrapper.BoolValue l4_packet = 167311273;
  ywrapper.BoolValue l4_source_port = 90182178;
  ywrapper.BoolValue outer_ip_proto = 317491232;
  ywrapper.BoolValue outer_tpid = 192373913;
  ywrapper.BoolValue outer_vlan_cfi = 286299834;
  ywrapper.BoolValue outer_vlan_dot1p = 318189580;
  ywrapper.BoolValue outer_vlan_id = 246371409;
  ywrapper.BoolValue pipeline_id = 215932380;
  ywrapper.BoolValue source_ipv4_address = 68858342;
  ywrapper.BoolValue source_mac_address = 335291014;
  ywrapper.BoolValue source_port = 73969881;
  ywrapper.BoolValue source_trunk = 431883282;
  ywrapper.BoolValue system_port_class = 536374618;
  ywrapper.BoolValue tos = 255672898;
  ywrapper.BoolValue tunnel_terminated = 473902200;
  ywrapper.BoolValue virtual_routing_and_forwarding_id = 297947686;
  ywrapper.BoolValue virtual_slice_group = 521852650;
  ywrapper.BoolValue vlan_inner_present = 336905570;
  ywrapper.BoolValue vlan_outer_present = 468273823;
}
